aptos-move/block-executor/src/counters.rs

```diff
@@ -46,6 +68,17 @@ pub static TASK_VALIDATE_SECONDS: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
+pub static WORK_WITH_TASK_SECONDS: Lazy<Histogram> = Lazy::new(|| {
```

Comment:
How about using one HistogramVec with different labels, given all of them are using the same bucket. This makes it easy to show everything in one graph.

Reply:
@grao1991 - That's a good point, but I don't show all the counters in one graph (because it becomes overwhelming), so I would like to leave them as separate histograms for clarity.